pkg/rtc/mediatrackreceiver.go

```diff
@@ -5,7 +5,6 @@ import (
 	"sort"
 	"strings"
 	"sync"
-	"time"
```

Comment:
Changes here are to move out the dynacast and media loss proxy stuff into their own modules and into top level `MediaTrack`.